.maintain/push-image.sh

```diff
@@ -6,24 +6,26 @@ target_tag=$2
 PROVIDER_BIN_NAME="dip-provider-node-template"
 CONSUMER_BIN_NAME="dip-consumer-node-template"
 
-# publish to docker hub
-docker tag local/standalone-node:$source_tag ${DOCKER_HUB_STANDALONE}:$target_tag
-docker tag local/kilt-node:$source_tag ${DOCKER_HUB_PARACHAIN}:$target_tag
-docker tag local/$PROVIDER_BIN_NAME:$source_tag ${DOCKER_HUB_DIP_PROVIDER_TEMPLATE}:$target_tag
-docker tag local/$CONSUMER_BIN_NAME:$source_tag ${DOCKER_HUB_DIP_CONSUMER_TEMPLATE}:$target_tag
+# Function to tag and push images
+tag_and_push() {
+  local source=$1
+  local target=$2
+  local registry=$3
 
-docker push ${DOCKER_HUB_STANDALONE}:$target_tag
-docker push ${DOCKER_HUB_PARACHAIN}:$target_tag
-docker push ${DOCKER_HUB_DIP_PROVIDER_TEMPLATE}:$target_tag
-docker push ${DOCKER_HUB_DIP_CONSUMER_TEMPLATE}:$target_tag
+  docker tag $source $target
+  docker push $target &
+}
 
-# publish to AWS
-docker tag local/standalone-node:$source_tag $AWS_REGISTRY/kilt/prototype-chain:$target_tag
-docker tag local/kilt-node:$source_tag $AWS_REGISTRY/kilt-parachain/collator:$target_tag
-docker tag local/$PROVIDER_BIN_NAME:$source_tag $AWS_REGISTRY/$PROVIDER_BIN_NAME:$target_tag
-docker tag local/$CONSUMER_BIN_NAME:$source_tag $AWS_REGISTRY/$CONSUMER_BIN_NAME:$target_tag
+# Tag and push to Docker Hub
+tag_and_push local/standalone-node:$source_tag ${DOCKER_HUB_STANDALONE}:$target_tag
```

Comment:
I only see two arguments here. Where do we pass the `registry`?

Reply:
`registry` is prefix of target tag

Reply:
Yes but the function takes three arguments, right? Here we're only passing two to `tag_and_push`: `local/standalone-node:$source_tag` and `${DOCKER_HUB_STANDALONE}:$target_tag`.

Reply:
removed 3rd param